src/Compilers/Core/Portable/Operations/IInvalidExpression.cs

```diff
@@ -13,7 +13,7 @@ public interface IInvalidExpression : IOperation
         /// <summary>
         /// Child operations.
         /// </summary>
-        ImmutableArray<IOperation> Children { get; }
```

Comment:
that is when all provide ImmutableArrray at the first place, for us, some uses ImmutableArray but some other uses yield return so, can't be consolidated to ImmutableArray